pkg/cli/aws/aws.go

```diff
@@ -165,13 +184,13 @@ func EnsureValidCredentials(ctx context.Context, copts *CredentialOptions) error
 		return nil
 	}
 
-	if os.Getenv("AWS_ACCESS_KEY_ID") != "" {
-		copts.Log.Debug("Skipping AWS credentials refresh check, AWS_ACCESS_KEY_ID is set")
-		return nil
+	if copts == nil {
+		copts = DefaultCredentialOptionsWithLog(nil)
```

Comment:
This doesn't need to change.

```suggestion
		copts = DefaultCredentialOptions()
```

Reply:
Adjusted!